### DISCUSSION TITLE
java/javaSE%E5%9F%BA%E7%A1%80/02_Java%E8%AF%AD%E8%A8%80%E5%9F%BA%E7%A1%80

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#933] [Announcements]
# java/javaSE%E5%9F%BA%E7%A1%80/02_Java%E8%AF%AD%E8%A8%80%E5%9F%BA%E7%A1%80

Java语言基础 一、关键字 1.1、是什么 定义：被Java语言赋予了特殊含义，用做专门用途的字符串（单词） 特点：关键字中所有的字母都是小写 常见关键字如下面两个表格所示。 imageimage imageimage 注意：以上关键字了解就可以，不需要刻意去记忆，后续大部分都会学到。 1.2、保留字 定义：现有Java版本尚未使用，但以后版本可能会...

http://localhost:8080/java/javaSE%E5%9F%BA%E7%A1%80/02_Java%E8%AF%AD%E8%A8%80%E5%9F%BA%E7%A1%80.html

**Comment by hjc1732** [comment]
不错
